### PR TITLE
Update webhooks.md

### PR DESCRIPTION
**Pull request by VeArnold**

### CODE REVIEW

File: content/configuration/webhooks.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Webhooks
+description: Set up webhooks for automatic builds
+weight: 6
+aliases: /building/webhooks
+---
+
+### Configuring webhooks with Codemagic
```
Comment (helinanever) [inline content/configuration/webhooks.md:8]: remove this heading

File: content/configuration/webhooks.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Webhooks are necessary in order to be able to trigger builds automatically in response to events in the repository.
+
+{{<notebox>}}
+The payload URL has the following format: `https://api.codemagic.io/hooks/[appId]`.
+
+You can find your app ID in the browser URL after `app/` when you open the app on Codemagic: `https://codemagic.io/app/[appId]`
+{{</notebox>}}
+
+#### Setting up webhooks for Github
```
Comment (helinanever) [inline content/configuration/webhooks.md:18]: all of these headings should be ##

File: content/configuration/webhooks.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+#### Setting up webhooks for Bitbucket
+
+Open your application repository, go to **Settings** > **Webhooks** (in **Workflow** section) > **Add webhook**, then enter an arbitrary title for the webhook and paste the **payload URL** in the **URL** field. For **Triggers**, select **Choose from a full list of triggers** and select the following events: **Push** in the **Repository** section and **Created**, **Updated**, **Merged** in the **Pull Request** section.
+
+#### Setting up webhooks for AWS CodeCommit
+
+In order to start using webhooks with **AWS CodeCommit**, it is first necessary to create a subscription with the **AWS Simple Notification Service**.
+
+##### Configuring the subscription
```
Comment (helinanever) [inline content/configuration/webhooks.md:34]: ###

File: content/configuration/webhooks.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Open up **AWS Simple Notification Service** in the **AWS Console**.
+2. Navigate to **Topics** > **Create topic**.
+3. Set the type to **Standard**, give the topic a name and click on **Create topic**.
+4. Navigate to **Subscriptions** > **Create subscription**.
+5. Select the previously configured topic, set the protocol to **HTTPS**, and set the Codemagic **payload URL** as the endpoint. Proceed by clicking **Create subscription**.
+6. In the Codemagic UI, navigate to your application and select the **Webhooks** tab.
+7. Under **Recent deliveries** select the most recent webhook, and copy the subscription link from under the **Results** tab to your browser.
+
+##### Configuring webhook events
```
Comment (helinanever) [inline content/configuration/webhooks.md:44]: This should be ###

File: content/configuration/webhooks.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Open your application repository and navigate to **Notify** > **Create notification rule** and enter a name for your Notification rule.
+
+Under **Events that trigger notifications**, select **Source updated** and **Created** events in the **Pull request** section and select the **Created** and **Updated** events in the **Branches and tags** section.
```
Comment (helinanever) [inline content/configuration/webhooks.md:48]:
```suggestion
Under **Events that trigger notifications**, select the **Source updated** and **Created** events in the **Pull request** section, and select the **Created** and **Updated** events in the **Branches and tags** section.
```

File: content/configuration/webhooks.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+---
+title: Webhooks
+description: Set up webhooks for automatic builds
+weight: 6
+aliases: /building/webhooks
```
Comment (helinanever) [inline content/configuration/webhooks.md:5]: this needs the previous alias too (from /flutter-configuration/webhooks)

Comment (VeArnold) [reply content/configuration/webhooks.md:5]: The alias actually already pointed to here.

**Review by helinanever — APPROVED** [review]
Lgtm

**Pull request merged**